spell/psm-checklists.md

```diff
@@ -0,0 +1,113 @@
+# PSM Checklists
+
+## PSM Migration Checklist
+
+- [ ] IF contracts are being on-boarded in the current spell, execute the full [LitePSM On-boarding Checklist](#litepsm-on-boarding-checklist)
+- [ ] IF a PSM is deprecated in the migration, execute the full [PSM Off-boarding Checklist](#psm-off-boarding-checklist)
+- [ ] Amount of `gem` being migrated matches the Exec Sheet
+- [ ] Collateral amount that is fetched from the source PSM matches the collateral is added to the destination PSM
+- [ ] No bad debt is left behind
+- Source PSM:
+    - [ ] ilk matches source PSM
+    - [ ] IF present, `tin` matches the value in the Exec Sheet
+    - [ ] IF present, `tout` matches the value in the Exec Sheet
+    - [ ] `AutoLine` is updated according to the Exec Sheet
+- Destination PSM:
+    - [ ] ilk matches destination PSM
+    - [ ] IF present, `tin` matches the value in the Exec Sheet
+    - [ ] IF present, `tout` matches the value in the Exec Sheet
+    - [ ] `AutoLine` is updated according to the Exec Sheet
+    - [ ] `buf` matches the value in the Exec Sheet
+    - [ ] `fill` is called to ensure liquidity is immediately available after spell execution
+
+## LitePSM On-boarding Checklist
+
+- Deployed Contracts
+    - `DssLitePsm`
+        - [ ] Uses the agreed best Solidity compiler version at the time
+        - [ ] Optimizer is **enabled**
+        - [ ] Contract has been reviewed by at least a couple of DAO dev teams
```

Comment:
- Couple sounds arbitrary, do you want to fix it by `minimum 2`?
- What is DAO dev team? "[Active Ecosystem Actor](https://mips.makerdao.com/mips/details/MIP101#2-8-2-active-ecosystem-actors)"?
- How do we check that it was reviewed by at least 2 teams in this specific example?

Reply:
I'll make some changes for clarity.

> How do we check that it was reviewed by at least 2 teams in this specific example?
I.e.: the reviewer could check the history of the PRs in the original repo and check if members of at least 2 different teams have approved it. Ofc this assumes the reviewer know who those teams are and which person belongs to each team.

Another idea could be to use attestations for that purpose, however I don't think it would be feasible to do it this time, as we'd have to convince all teams involved to use it. This can be discussed as a process improvement for the future.